04.INTERNALS.md

```diff
@@ -246,7 +246,7 @@ simple value is a pre-defined constant which can be:
 
 Compressed pointers were introduced to save heap space.
 
-![Compressed Pointer]({{ site.baseurl }}/img/ecma_compressed.png){: class="thumbnail center-block img-responsive" }
+![Compressed Pointer]({{ site.github.url }}/img/ecma_compressed.png){: class="thumbnail center-block img-responsive" }
```

Comment:
Yes, I am. You can check it here: http://bzsolt.github.io/jerryscript/internals/#compressed-pointers
It is also generated by the script.